src/command/schema/fetch.rs

```diff
@@ -0,0 +1,55 @@
+use anyhow::Result;
+use houston as config;
+use rover_client::blocking::Client;
+use rover_client::query::schema::get;
+use structopt::StructOpt;
+
+#[derive(Debug, StructOpt)]
+pub struct Fetch {
+    /// ID of the graph to fetch from Apollo Studio
+    #[structopt(name = "SCHEMA_ID")]
+    schema_id: String,
```

Comment:
this was kind of confusing to me as a user, since what we're really looking for here is the name of the graph in apollo studio. it is technically an ID, but in the UI it is titled `Graph Title`, not `Schema ID`. Might be worth updating this to reflect that verbiage

Reply:
Changing it to `graph_name` instead. I don't think `Title` makes much sense, and when creating a new graph in studio, it asks for a `graph name` instead. I know, the naming is a bit confusing there